interactive_ai/libs/iai_core_py/iai_core/entities/model_template.py

```diff
@@ -264,6 +264,10 @@ def __new__(cls, *args):
         17,
         TaskInfo(domain=Domain.KEYPOINT_DETECTION, is_trainable=True, is_anomaly=False, is_global=False, is_local=True),
     )
+    ANOMALY = (
```

Comment:
Same question, shouldn't the other anomaly types be removed? I am also a bit confused, this is a clean up, shouldn't this domain already exist before?